src/fuselage/CCPACSFuselageStringerFramePosition.cpp

```diff
@@ -16,22 +16,201 @@
 
 #include "CCPACSFuselageStringerFramePosition.h"
 
-namespace tigl
-{
+#include <Bnd_Box.hxx>
+#include <BRepBndLib.hxx>
+#include <gp_Pln.hxx>
+#include <BRepBuilderAPI_MakeFace.hxx>
+
+#include "generated/CPACSStringer.h"
+#include "tiglcommonfunctions.h"
+#include "CTiglError.h"
+#include "CNamedShape.h"
+#include "CCPACSFuselage.h"
+#include "CCPACSFrame.h"
+#include "CCPACSFuselageStringer.h"
+
+namespace tigl{
+// NOTE:
+// CPACS stores absolute positionings, from which we calculate cached relative ones
+// if relative positions are set, they are cached and immediately converted to absolute ones
+// if absolute positions are set, the cache is invalidated
+
 CCPACSFuselageStringerFramePosition::CCPACSFuselageStringerFramePosition(CCPACSFrame* parent, CTiglUIDManager* uidMgr)
-    : generated::CPACSStringerFramePosition(parent, uidMgr)
+    : generated::CPACSStringerFramePosition(parent, uidMgr) {}
+
+CCPACSFuselageStringerFramePosition::CCPACSFuselageStringerFramePosition(CCPACSFuselageStringer* parent, CTiglUIDManager* uidMgr)
+    : generated::CPACSStringerFramePosition(parent, uidMgr) {}
+
+gp_Pnt CCPACSFuselageStringerFramePosition::GetRefPoint() const {
+    return {
```

Comment:
I'll change this to return gp_Pnt(x,y,z), since this will not compile with vc 2008